microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,626 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/02/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
+This tutorial is for developers using WebView2 in immersive HoloLens 2 Unity applications.  WebView2 on HoloLens 2 and the WebView plugin for Unity are both in Preview and are subject to change before general availability.
+
+WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](/hololens/hololens-update-hololens).
+
+For WebView2-enabled 2D applications on HoloLens 2, see [Get started with WebView2 in WinUI 2 (UWP) apps](./winui2.md).
+
+
+In this tutorial, you:
+
+* Set up your development tools for creating HoloLens 2 Unity apps that use WebView2 to display web content.
+* Install the Mixed Reality Toolkit (MRTK) by using the Mixed Reality Feature Tool.
+* Create an initial Unity project for HoloLens 2 development.
+* Add the Microsoft Mixed Reality WebView plugin for Unity.
+* Configure a WebView2 prefab that displays webpage content in your HoloLens 2 app.
+* Learn about WebView2 concepts and interactions in Unity.
+
+
+<!-- ------------------------------ -->
+#### Completed project
+
+A completed version of this Getting Started project is available in the **WebView2Samples** repo. You can use the completed project (from the repo or from following the steps below) as a baseline to add more WebView2 functionality and other features to your HoloLens 2 Unity app.
+
+A completed version of this tutorial project is available in the **WebView2Samples** repo:
+
+* Sample name: **HoloLens2_GettingStarted**
+* Repo directory: [HoloLens2_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/HoloLens2_GettingStarted)
+* Unity project folder: **HoloLens2GetStartedApp**
+
+Follow the major Step sections in sequence, below.
+
+
+<!-- ------------------------------ -->
+#### About HoloLens 2, Unity, Mixed Reality Toolkit, and WebView2
+
+
+<!-- ------------------------------ -->
+###### HoloLens 2
+
+**HoloLens 2** is a groundbreaking, untethered holographic device, offering a hands-free and immersive mixed reality experience. HoloLens 2 seamlessly overlays digital information onto the real world, rendering high-definition holograms that stay in place and respond naturally to interaction.
+
+You use the Unity game engine and Visual Studio to create your HoloLens 2 project.
+
+
+<!-- ------------------------------ -->
+###### Unity
+
+**Unity**, a versatile game engine, is a popular choice for creating 3D applications, including mixed reality experiences for HoloLens 2 devices, due to its extensive features and robust capabilities for augmented reality (AR), virtual reality (VR), and mixed reality (MR).
+
+
+<!-- ------------------------------ -->
+###### Mixed Reality Toolkit (MRTK)
+
```

Comment:
@michaelfarnsworth - review table
* sequence of row
* Description column summary sentences.
* Goal: differentiate roles of the 3 things